Makefile

```diff
@@ -0,0 +1,44 @@
+.PHONY:	image publish
+
+ACCOUNT?=$(shell aws sts get-caller-identity | jq -r .Account)
+STAGE?=dev
+NAME?=$(shell awk -F: '$$1=="name" {print $$2}' deployment.yaml | sed -e 's/\s//g')
+ECR?=${ACCOUNT}.dkr.ecr.eu-west-1.amazonaws.com
+TAG?=$(shell if git describe > /dev/null 2>&1 ; then   git describe; else   git rev-parse --short HEAD; fi)
+
+IMAGE?=${NAME}/${STAGE}
+REPO?=${ECR}/${IMAGE}
+
+all: publish
+
+image: assets
+	@echo Building ${REPO}:${TAG} ...
+	@docker build --tag ${REPO}:${TAG} .
+	@echo Done.
+
+publish: image
+	@echo Publishing image: ${REPO}:${TAG} ...
+	@docker push ${REPO}:${TAG} 2>&1
+	@echo Done.
```

Comment:
I think you need a `.dockerfile`. Probably a minimum of:
```
.dockerignore
.git/
.github/
.gitignore
CHANGELOG.md
Dockerfile
Makefile
README.md
deployment.yaml
```

Reply:
Added.